lib/metadata/TableSchema.js

```diff
@@ -54,6 +57,14 @@ class TableSchema {
     return uriTemplate(url)
   }
 
+  parseNullValues () {
+    const tableGroupNullValues = RdfUtils.findValues(this.dataset, this.tableGroup, this.ns.null)
+    const tableNullValues = RdfUtils.findValues(this.dataset, this.table, this.ns.null)
+    const tableSchemaNullValues = RdfUtils.findValues(this.dataset, this.root, this.ns.null)
+
+    return [...tableGroupNullValues, ...tableNullValues, ...tableSchemaNullValues, '']
```

Comment:
Not sure about this here, but I'm guessing that `csvw:null` is additive, and that an empty cell should be treated at all times.

Might ensure uniqueness with `Set`?